doc/rippled-example.cfg

```diff
@@ -987,23 +972,6 @@ pool.ntp.org
 [ips]
 r.ripple.com 51235
 
-# Public keys of the validators that this rippled instance trusts.  The latest
```

Comment:
@nbougalis @ximinez While it does make it marginally harder to bootstrap a server, I'd like for the separate validators file to be the convention going forward.

So instead of leaving the validators and quorum sections commented out as a fallback, I'd like to make sure any relevant documentation is updated to include instructions to copy `validators-example.txt` in addition to `rippled-example.cfg`. Could you point me to any you are aware of? So far I've found this:
https://wiki.ripple.com/Setup_a_public_server

@ximinez What would `[validators_file]` be set to?
I'm thinking that if we always instruct people to copy `rippled.cfg` and `validators.txt` to the same directory, rippled will be able to find the validators file without needing to specify the location with `[validators_file]`.